### ISSUE TITLE
Missing refresh token

### ISSUE THREAD

**Issue by knfoo** [fydrah/loginapp#41]
**Describe the bug**
I have setup loginapp with helm with `refreshToken: "true"`
But I do not get a refresh token

**Loginapp & Dex version**
loginapp: `latest`
dex: `v2.26.0`

**Configuration**
I have not done a lot of configuration changes from the values.yaml file.

```
config:
  name: auth-login
  secret: "..."
  clientID: "login-cnp"
  clientSecret: "..."
  clientRedirectURL:
  issuerRootCA: # +doc-gen:break
    configMap:
    key: ca.pem
  issuerInsecureSkipVerify: true
  issuerURL: "https://dex/dex"
  refreshToken: "true"
  tls:
    enabled: false
    secretName:
    altnames: []
    altIPs: []
  # List of kubernetes clusters to add on web frontend
  clusters:
    - name: dc9-1
      server: https://k8s-api:6443
      certificate-authority: |
        -----BEGIN CERTIFICATE-----
        ...
        -----END CERTIFICATE-----
        #insecure-skip-tls-verify: false
```
Install Dex and connect it to AD with the ldap integration.

**Expected behavior**
I would expect to get a refresh token so that I do not need to login everyday.

If I run loginapp in debug I can tell that the scopes reflect that I would want offline access.
`{"level":"debug","msg":"request token with the following scopes: [openid profile email groups offline_access]`

And
```
Extra:{Scopes:[] AuthCodeOpts:map[]} OfflineAsScope:true CrossClients:[] Scopes:[openid profile email groups]}
``` 

I am not sure if I am missing something. 
I have read https://github.com/fydrah/loginapp/issues/11 but it seems that the config has changed somewhat since 2018.

**Comment by fydrah** [comment]
Hello @knfoo ,

Yes we have a problem... I checked and yes we log something like `request token with the following scopes: [openid profile email groups offline_access]` but in fact, the scope is not in the request URL:
```
{"level":"debug","msg":"auth code url: https://dex.172.18.0.2.nip.io:32000/auth?client_id=loginapp-local\u0026redirect_uri=https%3A%2F%2Floginapp.127.0.0.1.nip.io%3A8443%2Fcallback\u0026response_type=code\u0026scope=openid+profile+email+groups\u0026state=BQmLmbAJx5iURozJ8zgamAoWl92pVHYReqQFN36Y8vE%3D","time":"2021-02-02T17:39:12-05:00"}
```
I think I broke something during the last release. I check this and fix it asap

**Comment by fydrah** [comment]
Ok I found why... https://github.com/fydrah/loginapp/blame/master/pkg/client/client.go#L51

Should be `c.Scopes` (which are prepared and should include offline_access scope) instead of `c.Config.Scopes`.

As a quick fix from your side, you can change the following option:
```
configOverwrites:
  oidc:
    scopes: ["openid", "profile", "email", "groups", "offline_access"] 
```

It will work until I'll fix it and publish the new image

**Comment by knfoo** [comment]
Great that you found the root cause and thanks for the workaround :+1:

**Comment by fydrah** [comment]
Released in [v3.2.2](https://github.com/fydrah/loginapp/releases/tag/v3.2.2)
